src/commands/rename_file.ts

```diff
@@ -0,0 +1,63 @@
+import { CLICommand, ParametersHelper } from '../CLICommand';
+import {
+	BoostParameter,
+	DryRunParameter,
+	FileIdParameter,
+	DrivePrivacyParameters,
+	FileNameParameter
+} from '../parameter_declarations';
+import { cliArDriveFactory } from '..';
+import { SUCCESS_EXIT_CODE } from '../CLICommand/error_codes';
+import { CLIAction } from '../CLICommand/action';
+import { EID, Wallet } from 'ardrive-core-js';
+
+new CLICommand({
+	name: 'rename-file',
+	parameters: [FileIdParameter, FileNameParameter, BoostParameter, DryRunParameter, ...DrivePrivacyParameters],
+	action: new CLIAction(async function action(options) {
+		const parameters = new ParametersHelper(options);
+
+		const dryRun = !!parameters.getParameterValue(DryRunParameter);
```

Comment:
We do this double negation dance in so many places that I think it's time for a `isDryRun` function on ParametersHelper.

Reply:
Implements isDryRun and adds a couple of tests: 8306eac